src/deadline_test_fixtures/fixtures.py

```diff
@@ -444,23 +447,52 @@ def worker_config(
         ), f"Expected exactly one Worker agent whl path, but got {resolved_whl_paths} (from pattern {worker_agent_whl_path})"
         resolved_whl_path = resolved_whl_paths[0]
 
-        if operating_system.name == "AL2023":
+        if operating_system.is_amazon_linux():
             dest_path = posixpath.join("/tmp", os.path.basename(resolved_whl_path))
-        else:
+        elif operating_system.is_windows():
             dest_path = posixpath.join(
                 "C:\\Windows\\System32\\Config\\systemprofile\\AppData\\Local\\Temp",
                 os.path.basename(resolved_whl_path),
             )
+        else:
+            raise NotImplementedError(f"Unsupported operating system: {operating_system.name}")
         file_mappings = [(resolved_whl_path, dest_path)]
 
-        LOG.info(f"The whl file will be copied to {dest_path} on the Worker environment")
-        worker_agent_requirement_specifier = dest_path
+        LOG.info(
+            f"The worker agent whl file will be copied to {dest_path} on the Worker environment"
+        )
+        pip_install_requirement_specifiers.append(dest_path)
     else:
         worker_agent_requirement_specifier = os.getenv(
             "WORKER_AGENT_REQUIREMENT_SPECIFIER",
             "deadline-cloud-worker-agent",
         )
         LOG.info(f"Using Worker agent package {worker_agent_requirement_specifier}")
+        pip_install_requirement_specifiers.append(worker_agent_requirement_specifier)
+
+    openjd_sessions_whl_path = os.getenv("OPENJD_SESSIONS_WHL_PATH")
+    if openjd_sessions_whl_path:
+        LOG.info(f"Using openjd-sessions agent whl file: {openjd_sessions_whl_path}")
+        resolved_whl_paths = glob.glob(openjd_sessions_whl_path)
+        assert (
+            len(resolved_whl_paths) == 1
+        ), f"Expected exactly one openjd-sessions whl path, but got {resolved_whl_paths} (from pattern {openjd_sessions_whl_path})"
+        resolved_whl_path = resolved_whl_paths[0]
+
+        if operating_system.is_amazon_linux():
+            dest_path = posixpath.join("/tmp", os.path.basename(resolved_whl_path))
+        elif operating_system.is_windows():
+            dest_path = posixpath.join(
+                "C:\\Windows\\System32\\Config\\systemprofile\\AppData\\Local\\Temp",
+                os.path.basename(resolved_whl_path),
+            )
+        else:
+            raise NotImplementedError(f"Unsupported operating system: {operating_system.name}")
+        file_mappings.append((resolved_whl_path, dest_path))
+        LOG.info(
+            f"The openjd-sessions whl file will be copied to {dest_path} on the Worker environment"
+        )
+        pip_install_requirement_specifiers.append(dest_path)
```

Comment:
nit - this logic looks very similar to the worker agent whl install, can there be a helper to reduce duplicated code?